examples/slab2.py

```diff
@@ -0,0 +1,32 @@
+"""
+Slab2 - A Comprehensive Subduction Zone Geometry Model
+======================================================
+
+Slab2 is a three-dimensional compilation of global subduction geometries, separated
+into regional models for each major subduction zone.
+More information at the
+`USGS <https://www.sciencebase.gov/catalog/item/5aa1b00ee4b0b1c392e86467>`__ website.
+"""
+
+import rockhound as rh
+import matplotlib.pyplot as plt
+import cmocean
+import cartopy.crs as ccrs
+
+# Load a geometry of the Alaska subduction zone
+grid = rh.fetch_slab2(zone="south_america")
```

Comment:
The zone in the comment does not corresponds with the actual fetched zone.

Reply:
Done